source/query-bar.txt

```diff
@@ -57,7 +57,7 @@ operator.
    Builder to enter a query into the query bar. For details, see
    :ref:`build-query`.
 
-As you type, the :guilabel:`Apply` button is disabled and the
+As you type, the :guilabel:`Find` button is disabled and the
```

Comment:
I believe that is the reason the text is using the full element name: to distinguish between the Find button, which is disabled, and the Filter label, which does in fact turn red after starting out black.  Also note that these two actions simultaneously - "As you type ... " - so, I'm concerned that breaking this short sentence in two offers more danger to comprehension than potential  value.